code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -71,12 +71,23 @@
 	return brainloss
 
 //These procs fetch a cumulative total damage from all limbs
-/mob/living/carbon/human/getBruteLoss(organic_only=0)
-	var/amount = 0
-	for(var/datum/limb/O in limbs)
-		if(!(organic_only && O.limb_status & LIMB_ROBOT))
-			amount += O.brute_dam
-	return amount
+/mob/living/carbon/human/getBruteLoss(organic_only = FALSE, external_only = FALSE)
+	. = 0
+	for(var/i in limbs)
+		var/datum/limb/bodypart = i
+		if(organic_only && bodypart.limb_status & LIMB_ROBOT)
```

Comment:
if this is a heavily used proc then put these ifs outside the loop and having multiple copies of the loop code, dumb looking but better performing (slightly)

Reply:
I'll make a new proc. This is a hot one.